src/weights.jl

```diff
@@ -475,13 +559,22 @@ function wsum(A::AbstractArray{T}, w::AbstractVector{W}, dim::Int) where {T<:Num
     _wsum!(similar(A, wsumtype(T,W), Base.reduced_indices(axes(A), dim)), A, w, dim, true)
 end
 
+function wsum(A::AbstractArray{<:Number}, w::UnitWeights{S,T}, dim::Int) where {S,T}
+    (size(A, dim) != length(w)) && throw(DimensionMismatch("Inconsistent array dimension."))
```

Comment:
No need for parentheses:
```suggestion
    size(A, dim) != length(w) && throw(DimensionMismatch("Inconsistent array dimension."))
```
Same elsewhere.

Reply:
You know, I got errors if I didn't use parentheses in earlier versions of Julia, so I've just used them ever since. They don't seem necessary any longer though.

Reply:
They're still there. ;-)

Reply:
Zut. I tried to be more thorough this time.